iamap.js

```diff
@@ -116,10 +117,10 @@ async function create (store, options, map, depth, data) {
  */
 async function load (store, id, depth = 0, options) {
   // depth and options are internal arguments that the user doesn't need to interact with
-  if (depth !== 0 && typeof options !== 'object') {
+  if (depth !== 0 && (typeof options !== 'object' || options == null)) {
```

Comment:
This is I think fixing a bug when `null` is passed, since `typeof null === 'object'`.